WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
+
+RSpec.describe "Competition WCIF" do
+  let!(:competition) {
+    FactoryGirl.create(
+      :competition,
+      :with_delegate,
+      id: "TestComp2014",
+      name: "Test Comp 2014",
+      start_date: "2014-02-03",
+      end_date: "2014-02-05",
+      external_website: "http://example.com",
+      showAtAll: true,
+      event_ids: %w(333 444),
+    )
+  }
+  let(:delegate) { competition.delegates.first }
+  let(:sixty_second_2_attempt_cutoff) { Cutoff.new(numberOfAttempts: 2, attemptValue: ONE_MINUTE_IN_CENTISECONDS) }
+  let(:top_16_advance) { AdvanceToNextRoundRequirement.new(type: "ranking", ranking: 16) }
+  let!(:round333_1) { FactoryGirl.create(:round, competition: competition, event_id: "333", number: 1, cutoff: sixty_second_2_attempt_cutoff, advance_to_next_round_requirement: top_16_advance) }
+  let!(:round333_2) { FactoryGirl.create(:round, competition: competition, event_id: "333", number: 2) }
+  let!(:round444_1) { FactoryGirl.create(:round, competition: competition, event_id: "444", number: 1) }
+  before :each do
+    competition.reload
+  end
+
+  context "to_wcif" do
```

Comment:
This is a matter of taste, but I think that `describe` reads better when we are testing *some particular entity - like method*. Not sure how much is it used in our code, but there's a convention to prefix instance/class methods with `#`/`.` accordingly.

General comment of a small importance:
As you placed the file within model specs I'd suggest the following structure (so we still describe the actual model):

```ruby
RSpec.describe Competition do
  context "WCIF" do
    describte "#to_wcif" do
      # ...
    end

    describte "#load_wcif!" do
      # ...
    end
  end
end
```

It may be me just inventing strange things, so feel free to ignore this one completely.
Still, would like to hear @larspetrus' opinion.